modules/scroll_snap.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Scroll snap</title>
+    <style>
+      li {
```

Comment:
editor warning here about empty ruleset in case we want to add something in there, defaults, maybe?

Reply:
It's a JS hook.